tests/manuscripts/example/content/metadata.yaml

```diff
@@ -9,13 +9,16 @@ author_info:
     initials: DSH
     orcid: 0000-0002-3012-7446
     twitter: dhimmel
-    affiliations: Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania
+    affiliations:
+      - Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania
+      - Department of Biological & Medical Informatics, University of California, San Francisco
     funders: GBMF4552
   -
     github: agitter
     name: Anthony Gitter
     initials: AG
     orcid: 0000-0002-5324-9833
     twitter: anthonygitter
-    affiliations: Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison and Morgridge Institute for Research
+    affiliations:
+      - Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison and Morgridge Institute for Research
```

Comment:
We can split my affiliations as well to avoid confusion with the `and`:
```yaml
affiliations:
       - Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison
       - Morgridge Institute for Research
```